### PR TITLE
Consume UserChallenge amount instead of using config

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Make the discovery nodes the source of truth for reward amounts.

### Dragons

N/A

### How Has This Been Tested?

Tested manually against stage

### How will this change be monitored?

N/A

### CODE REVIEW

File: src/common/models/AudioRewards.ts [hunk 1]
```diff
@@ -79,3 +80,15 @@ export type FlowSessionEvent =
   | FlowSessionResumeEvent
   | FlowSessionPassEvent
   | FlowSessionFailEvent
+
+/**
+ * @deprecated amounts should be pulled in directly from user challenges instead
```
Comment (rickyrombo) [inline src/common/models/AudioRewards.ts:85]: Still need config for the notifications.

Moved this here so at least notifications can share one source of truth between web/mobile

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-consume-amount

**Review by Kyle-Shanks — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-consume-amount

File: src/common/models/AudioRewards.ts (hunk 1 above)
Comment (piazzatron) [reply src/common/models/AudioRewards.ts:85]: nice - might be nice to include this context in the comment so we remember why we're keeping this around

**Review by piazzatron — APPROVED** [review]

**Pull request merged**